internal/server/shardGather.go

```diff
@@ -0,0 +1,8 @@
+package server
```

Comment:
It appears this file is not needed. Can we remove this?

Reply:
This file I have kept empty to add gather logic for multishareded commands later on. for now we can remove it but later for a clean code we can create the file again for all gather functions

Reply:
Yah an empty file seems a bit awkward. Lets remove it and add it back when needed.